src/components/navigation.js

```diff
@@ -1,17 +1,23 @@
 import { Link, Outlet } from 'react-router-dom';
+import '../styles/navigation.css';
 
 const Navigation = () => (
-  <header className="header">
-    <div className="logo">
-      <h1>Bookstore CMS</h1>
-    </div>
-    <nav className="navBar">
-      <Link to="/">BOOK</Link>
-      <Link to="categories">Category</Link>
-    </nav>
-    <div className="profile" />
+  <>
+    <header className="header">
+      <div className="logo">
+        <h1>Bookstore CMS</h1>
+      </div>
+      <nav className="navBar">
+        <Link className="link1" to="/">BOOK</Link>
+        <Link className="link2" to="categories">Category</Link>
+      </nav>
+      <div className="profile">
+        <svg xmlns="http://www.w3.org/2000/svg" fill="#0290ff" width="1.5rem" height="1.5rem" viewBox="0 0 32 32" version="1.1"><path d="M4 28q0 0.832 0.576 1.44t1.44 0.576h20q0.8 0 1.408-0.576t0.576-1.44q0-1.44-0.672-2.912t-1.76-2.624-2.496-2.144-2.88-1.504q1.76-1.088 2.784-2.912t1.024-3.904v-1.984q0-3.328-2.336-5.664t-5.664-2.336-5.664 2.336-2.336 5.664v1.984q0 2.112 1.024 3.904t2.784 2.912q-1.504 0.544-2.88 1.504t-2.496 2.144-1.76 2.624-0.672 2.912z" /></svg>
+      </div>
```

Comment:
- [ ] Please align the user icon to the right so the book cards are all aligned together from top to bottom just as in the original design. 

<img width="1721" alt="Screenshot 2023-05-13 at 3 43 12 PM" src="https://github.com/anbehindY/book-store/assets/50941074/f60e983f-aab1-47d9-8614-6b66d82d3c85">